tools/nni_trial_tool/trial_keeper.py

```diff
@@ -103,6 +104,32 @@ def main_loop(args):
 def trial_keeper_help_info(*args):
     print('please run --help to see guidance')
 
+def check_version(args):
+    trial_keeper_version = None
+    try:
+        trial_keeper_version = pkg_resources.get_distribution('nni').version
+    except:
+        #package nni does not exist
+        pass
+    if trial_keeper_version is None:
+        try:
+            trial_keeper_version = pkg_resources.get_distribution('nni-tool').version
+        except:
+            #package nni-tool does not exist
+            pass
+    if trial_keeper_version is None:
```

Comment:
"if not trial_keeper_version"？（pylint suggests...)

Reply:
changed to `if not trial_keeper_version`